Makefile

```diff
@@ -0,0 +1,64 @@
+#
+# Tools
+#
+ESLINT		:= ./node_modules/.bin/eslint
+JSCS		:= ./node_modules/.bin/jscs
+MOCHA       := ./node_modules/.bin/mocha
+ISTANBUL    := ./node_modules/.bin/istanbul
+COVERALLS   := ./node_modules/.bin/coveralls
+NPM		    := npm
+
+#
+# Files
+#
+GIT_HOOK_SRC   = '../../tools/githooks/pre-push'
+GIT_HOOK_DEST  = '.git/hooks/pre-push'
+LIB_FILES  	   = './lib'
+TEST_FILES     = './test'
+COVERAGE_FILES = './coverage'
+LCOV           = './coverage/lcov.info'
+
+#
+# Targets
+#
+.PHONY: all
+all:
+	$(NPM) install
```

Comment:
make npm install its own task, and make the other tasks depend on npm install. The nice thing about Make is that if node_modules already exists then it will not re run npm i. Look at this example here: https://github.com/NetflixUI/nodequark/blob/master/Makefile#L61-L62

node_modules depends on package.json. This means that npm i will only re-run if package.json has changed